### ISSUE TITLE
Empty commits

### ISSUE THREAD

**Issue by rawsome24** [QasimWani/LeetHub#213]
From last few days this code is doing empty commits. It is committing to repo even when zero file are changed, so please take a look and fix it.
See this for reference
![image](https://user-images.githubusercontent.com/95746772/152667194-84d938f9-2c6d-421b-a40a-e1454942a184.png)

**Comment by QasimWani** [comment]
Yeah, just pushed a patch on chrome. Should take 2-3 business days for Chrome to approve it.
Apologies for the disruptive experience.

**Comment by rawsome24** [comment]
Thanks Qasim!😃
